setup.py

```diff
@@ -15,6 +15,7 @@
         'console_scripts': [
             'start-crawl = sh_scrapy.crawl:main',
             'list-spiders = sh_scrapy.crawl:list_spiders',
+            'shub-image-info = sh_scrapy.crawl:shub_image_info',
```

Comment:
That's how all scripts in this package work. Also the package can be pip installed from source and it'll works.